### PR TITLE
Update DG Diagrams and Include TagCommand, OpenCommand Implementation

### LINKED ISSUE #139: Update the UML diagrams in DG

### LINKED ISSUE #155: Update DG to reflect the changes in HelloFile

### PR DESCRIPTION
**Pull request by luo-git**
Deleted proposed redo/undo implementations and manual UI testing instructions.

### CODE REVIEW

**Review by HynRidge — APPROVED** [review]
Can't really check the UML..but LGTM!

**Comment by luo-git** [comment]
> Can't really check the UML..but LGTM!

You can use "view file" at the top right corner to see the markdown file.

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -553,57 +486,3 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 * **GUI**: Graphical user interface
 * **Relative File Path**: A path that is relative to a current directory, it is combined with another file path to access a file.
 * **Absolute File Path**: The complete details required to locate the file or folder, starting with the root element.
-
---------------------------------------------------------------------------------------------------------------------
-
-## **Appendix: Instructions for manual testing**
```
Comment (anthony6401) [inline docs/DeveloperGuide.md:559]: Are we deleting appendix for manual testing altogether or will we add it later?

**Review by anthony6401 — APPROVED** [review]
LGTM! Good job on changing the DG.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (luo-git) [reply docs/DeveloperGuide.md:559]: I don't think we will be doing manual testing anymore. Moreover, this appendix is for us to read and probably not suitable to be included in DG.

**Pull request merged**